test/integration/patients/patient/patient-flow.js

```diff
@@ -1852,69 +1865,84 @@ context('patient flow page', function() {
   specify('actions with work:owned:manage permission', function() {
     cy
       .routeCurrentClinician(fx => {
-        fx.data.relationships.role = { data: { id: '66666' } };
+        fx.data = getCurrentClinician({
+          relationships: {
+            role: getRelationship(roleNoFilterEmployee),
+          },
+        });
+
         return fx;
       })
       .routeFlowActions(fx => {
-        fx.data = _.sample(fx.data, 4);
-
-        fx.data[0].id = '1';
-        fx.data[0].relationships.state = { data: { id: '22222' } };
-        fx.data[0].relationships.owner = { data: { id: '11111', type: 'clinicians' } };
-        fx.data[0].relationships.form = { data: { id: '11111' } };
-        fx.data[0].attributes.name = 'First In List';
-        fx.data[0].attributes.due_date = testDateAdd(5);
-        fx.data[0].attributes.sequence = 0;
-        fx.data[0].relationships.flow = { data: { id: '1' } };
-
-        fx.data[1].id = '2';
-        fx.data[1].relationships.state = { data: { id: '22222' } };
-        fx.data[1].relationships.owner = { data: { id: '11111', type: 'clinicians' } };
-        fx.data[1].attributes.name = 'Last In List';
-        fx.data[1].attributes.due_date = testDateAdd(5);
-        fx.data[1].attributes.sequence = 3;
-        fx.data[1].relationships.flow = { data: { id: '1' } };
-
-        fx.data[2].id = '3';
-        fx.data[2].attributes.sequence = 2;
-        fx.data[2].attributes.due_date = testDateAdd(5);
-        fx.data[2].attributes.due_time = null;
-        fx.data[2].relationships.flow = { data: { id: '1' } };
-        fx.data[2].relationships.form = { data: { id: '11111' } };
-        fx.data[2].relationships.owner = { data: { id: '11111', type: 'teams' } };
-        fx.data[2].relationships.state = { data: { id: '33333' } };
-
-        fx.data[3].id = '4';
-        fx.data[3].attributes.sequence = 1;
-        fx.data[3].attributes.due_date = null;
-        fx.data[3].attributes.due_time = null;
-        fx.data[3].relationships.flow = { data: { id: '1' } };
-        fx.data[3].relationships.owner = { data: { id: '11111', type: 'teams' } };
-        fx.data[3].relationships.state = { data: { id: '33333' } };
+        fx.data = [
+          getAction({
+            attributes: {
+              name: 'First In List',
+              due_date: testDateAdd(5),
+              sequence: 0,
+            },
+            relationships: {
+              flow: getRelationship(testFlow),
+              state: getRelationship(stateTodo),
+              owner: getRelationship('11111', 'clinicians'),
```

Comment:
`getCurrentClinician`

Reply:
Updated three of these in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/95fdae5462810ae830c55e4bdd8ff2ff3ed711ad.